lib/influxer/metrics/metrics.rb

```diff
@@ -45,6 +46,12 @@ def attributes(*attrs)
         end
       end
 
+      def values(*attrs)
```

Comment:
I think it's better to leave old _attributes_ name, to avoid incompatible changes.